flow-typed/npm/jest_v26.x.x.js

```diff
@@ -259,6 +259,7 @@ type DomTestingLibraryType = {
   // 5.x
   toHaveDisplayValue(value: string | string[]): void,
   toBeChecked(): void,
+  toHaveDescription(value: string | string[]): Result,
```

Comment:
It's part of https://github.com/testing-library/jest-dom/releases/tag/v5.6.0 and we use `"@testing-library/jest-dom": "^5.7.0",`